### PR TITLE
#166185725 Add slack integration for Travis CI

### PR DESCRIPTION
**Pull request by Rhytah**
#### What does this PR do?
Add slack integration for Continuous Integrations
#### Description of Task to be completed?

- [x] Add Github configurations to slack.

- [x] Add Pivotal Tracker configurations to slack.

- [x] Add Travis CI configurations to slack.

- [x] Add notification settings to `.travis.yml` file

#### Any background context you want to provide?
- Travis CI is a continuous integration platform that takes care of running your software tests and deploying your apps.
- This integration will allow our team to receive notifications in Slack.

#### What are the relevant pivotal tracker stories?
[#166185725](https://www.pivotaltracker.com/story/show/166185725)

#### Screenshots 
Team slack channel having Github, Pivotal Tracker and TRAVIS CI integrations running.
Screenshot 2019-05-28 at 10 56 08

### CODE REVIEW

**Review by Manorlds-Eaglespark — CHANGES_REQUESTED** [review]
Kindly edit your commit messages by adding a footer. Review the commits section in this workflow guide.
https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices

File: .travis.yml [hunk 1]
```diff
@@ -65,3 +65,7 @@ script:
 
   # Code coverage
   - coverage report
+
+  # slack integration for Travis builds
+notifications:
+  slack: andela:hHgpX64Ycz7RvLVk8Z8BESUo
```
Comment (reifred) [inline .travis.yml]: However, I request you to use environment variables at this point as this is sensitive data that should not be exposed.
I found a nice resource for setting environment variables in travis here. Make use of it.
https://docs.travis-ci.com/user/environment-variables/
Or encrypting them as in this guideline.
https://stackoverflow.com/questions/33735992/travis-ci-using-repository-environment-variables-in-travis-yml

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks @Rhytah, for the good work done.

File: .travis.yml (hunk 1 above)
Comment (Rhytah) [reply .travis.yml]: Thanks, @reifred for pointing this out. Let me get to protecting it.

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Review by reifred — APPROVED** [review]

**Review by Emmanuel-Dominic — APPROVED** [review]

**Pull request merged**